app/views/admin/bulk_uploads/new.html.erb

```diff
@@ -1,11 +1,6 @@
 <% content_for :page_title, "Bulk upload for: #{@edition.title}" %>
 <% content_for :title, "Bulk upload" %>
 <% content_for :context, "Attachments for #{@edition.format_name}" %>
-<% content_for :back_link do %>
```

Comment:
Could you also remove the back link from the set_titles template?

Reply:
Good shout. Have done